setup.py

```diff
@@ -112,7 +112,7 @@ def check_setuptools_features():
     zip_safe=False,
     python_requires='>=3.6',
     classifiers=[
-        'Development Status :: 4 - Beta',
+        'Development Status :: 5 - Beta',
```

Comment:
This line should stay 'Development Status :: 4 - Beta'. You can find a list of all the allowed classifier values at:

https://pypi.org/pypi?%3Aaction=list_classifiers